cli/api/daemon.go

```diff
@@ -182,7 +182,7 @@ func (d *daemon) stopISVCS() {
 	glog.Infof("isvcs shut down")
 }
 
-func (d *daemon) startRPC() {
+func (d *daemon) startRPC() error {
```

Comment:
I don't see a case that this method ever returns an error, and there are no checks for errors when we call `startRPC()`, so seems like this should be a void return value.